rules/prefer-starts-ends-with.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+const doesNotContain = (string, chars) => chars.every(char => !string.includes(char));
+
+const isSimpleString = string => doesNotContain(
+	string,
+	['^', '$', '+', '[', '{', '(', '\\', '.', '?', '*']
```

Comment:
@firasdib yeah, thank you. I was under the same impression so I asked for help to see if I missed something.

@sindresorhus, @jfmengels I am not sure if there is much I can add here. Please let me know if there is anything else I can do. Thanks.